### ISSUE TITLE
Collision detection on heatmap view

### ISSUE THREAD

**Issue by yoursweater** [mapbox/mapbox-gl-js#7116]
I'm not sure if this is a bug or working as intended, but essentially when I make a heatmap on a bunch of points and attempt to create a heatmap based on a number property on each of those points, I run into this effect where each point "removes" itself from the screen when I'm too zoomed out. This is the opposite effect of what I'm desiring, since the heatmap only works if every point contributes to determining density.

To be specific and put it in plain english, I'm taking the centroid of each zipcode within a city, and then appending a number of visitors to those points, and I'm trying to create a heatmap that shows which areas have the most visitors, but collision detection or something to that effect is preventing all points from being displayed at once.

**Comment by mourner** [comment]
Can you please share a minimal JSFiddle/JSBin/Codepen test case that demonstrates the behavior that you're describing? Please don't remove the issue template when submitting bug reports — they're there for a reason.

**Comment by mollymerp** [comment]
Going to close here – if you can reproduce the behavior you're seeing on jsbin or jsfiddle please open a new issue.
